src/gitlab.coffee

```diff
@@ -60,6 +60,16 @@ module.exports = (robot) ->
   trim_commit_url = (url) ->
     url.replace(/(\/[0-9a-f]{9})[0-9a-f]+$/, '$1')
 
+  strip_content = (str) ->
+    str.split(/\n/).filter (line, index) ->
+      # Remove the \n first so we have content only
+      if line.length != 0
+          line
+    .filter (_line, _index) ->
+      if _index <= 4
+        _line
+    .join("\n")
```

Comment:
@spruce sure we could do that. but as I said there can be `\n` (newline) elements in the array. so to distinguish that we need to loop over it.

I'm not a javascript expert. but if you have a better I'm all ears. :)